### PR TITLE
Cleanup to skip/flaky attributes

### PR DESCRIPTION
**Pull request by jkotalik**
- Moves SkipOnHelix to M.A.Testing
- Adds SkipOnCI
- Change namespaces to have everything in the same namespace (FlakyOn and FlakyAttribute were in a different namespace).

This is a testing only change.

### CODE REVIEW

File: src/Hosting/Hosting/test/Internal/ConsoleLifetimeTests.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Extensions.Hosting.Internal
+{
+    public partial class HostTests
+    {
+        [Fact]
+        public async Task ConsoleLifetimeDoesNotThrowExceptionFromProcessExit()
```
Comment (BrennanConroy) [inline src/Hosting/Hosting/test/Internal/ConsoleLifetimeTests.cs]: ?

File: src/TestingUtils/Microsoft.AspNetCore.Testing/src/xunit/SkipOnCIAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Testing
+{
+    /// <summary>
+    /// Skip test if running on helix (or a particular helix queue).
```
Comment (BrennanConroy) [inline src/TestingUtils/Microsoft.AspNetCore.Testing/src/xunit/SkipOnCIAttribute.cs]: Fix comment

File: src/Hosting/Hosting/test/Internal/ConsoleLifetimeTests.cs (hunk 1 above)
Comment (jkotalik) [reply src/Hosting/Hosting/test/Internal/ConsoleLifetimeTests.cs]: idk what happened here.

**Comment by jkotalik** [comment]
🆙 📅

**Comment by jkotalik** [comment]
@aspnet/build there will be build issues in the aspnetcore repo once this is merged. Please assignment me if I haven't started fixing it myself.

**Pull request merged**